Lib/test/test_unittest/test_async_case.py

```diff
@@ -434,6 +434,30 @@ async def cleanup(self, fut):
         test.doCleanups()
         self.assertEqual(events, ['asyncSetUp', 'test', 'cleanup'])
 
+    def test_setup_get_event_loop(self):
+        # See https://github.com/python/cpython/issues/95736
+        events = []
+        class TestCase1(unittest.IsolatedAsyncioTestCase):
+            def setUp(self):
+                self.loop = asyncio.get_event_loop_policy().get_event_loop()
+
+            async def test_demo1(self):
+                events.append('called demo1')
+
+
+        class TestCase2(unittest.IsolatedAsyncioTestCase):
+            def setUp(self):
+                self.loop = asyncio.get_event_loop_policy().get_event_loop()
+
+            async def test_demo2(self):
+                events.append('called demo2')
+
+        test = TestCase1('test_demo1')
+        test.debug()
+        self.assertEqual(events, ['called demo1'])
+        test = TestCase2('test_demo2')
+        test.debug()
+        self.assertEqual(events, ['called demo1', 'called demo2'])
```

Comment:
So comparing this to @graingert's repro from #95736, the key thing here is that running two test cases doesn't work in 3.11? And #95900 doesn't fix that?

Reply:
Yes https://github.com/python/cpython/pull/95900 is for https://github.com/python/cpython/issues/95899 which I separated to avoid confusion. This currently fails without the fix.

Reply:
Since (apart from the tests) there is only one line of code in this PR, I don't follow how it could be simplified. I guess I'll wait until #95900 lands.